azext_iot/deviceupdate/_help.py

```diff
@@ -0,0 +1,252 @@
+        type: command
+        short-summary: Create a Device Update account.
+        long-summary: This command may also be used to update an existing account provided the generic update options
+          in the update command are not needed.
+
+        examples:
+        - name: Create a Device Update account in target resource group using the resource group location.
+          text: >
+            az iot device-update account create -n {account_name} -g {resouce_group}
+
+        - name: Create a free sku Device Update account in target resource group with specified location and tags without polling.
+          text: >
+            az iot device-update account create -n {account_name} -g {resouce_group} -l westus --tags a=b c=d --sku Free --no-wait
+
+        - name: Create a Device Update account in target resource group with a system managed identity.
+          text: >
+            az iot device-update account create -n {account_name} -g {resouce_group} --assign-identity [system]
+
+        - name: Create a Device Update account in target resource group with a system managed identity then
+                assign the system identity to one or more scopes (space-separated) with the role of Contributor.
+          text: >
+            az iot device-update account create -n {account_name} -g {resouce_group} --assign-identity [system]
+            --scopes "/subscriptions/a12345ea-bb21-994d-2263-c716348e32a1/resourceGroups/ProResourceGroup/providers/Microsoft.Storage/storageAccounts/myStorageAccount"
+
+        - name: Create a Device Update account in target resource group with system managed and user-assigned managed identities then
+                assign the system identity to one or more scopes with a custom specified role.
+          text: >
+            az iot device-update account create -n {account_name} -g {resouce_group}
+            --assign-identity [system] "/subscriptions/a12345ea-bb21-994d-2263-c716348e32a1/resourcegroups/ProResourceGroup/providers/Microsoft.ManagedIdentity/userAssignedIdentities/myIdentity"
+            --scopes "/subscriptions/a12345ea-bb21-994d-2263-c716348e32a1/resourceGroups/ProResourceGroup/providers/Microsoft.Storage/storageAccounts/myStorageAccount"
+            --role "Storage Blob Data Contributor"
+    """
+
+    helps["iot device-update account update"] = """
+        type: command
+        short-summary: Update a Device Update account.
+
+        examples:
+        - name: Set a specific account tag attribute.
+          text: >
+            az iot device-update account update -n {account_name} --set tags.env='test'
+    """
+
+    helps["iot device-update account show"] = """
+        type: command
+        short-summary: Show the details of a Device Update account.
+
+        examples:
+        - name: Show a target account.
+          text: >
+            az iot device-update account show -n {account_name}
+
+        - name: Show a target account filtering on a specific property.
```

Comment:
query isnt jmes query?

Reply:
Not sure what you mean as this works perfectly fine.

![image](https://user-images.githubusercontent.com/28658112/172252170-7ee39123-7b38-45fc-b956-6d94cc7769a7.png)

![image](https://user-images.githubusercontent.com/28658112/172252284-84cc4fdb-5579-4c9f-9192-74221956c2b7.png)

Yes if identity is null / None you will not get anything back.